projects/ion/src/lib/avatar/avatar.component.html

```diff
@@ -0,0 +1,13 @@
+<div data-testid="ion-avatar" [class]="'ion-avatar ion-avatar--' + size">
```

Comment:
Since using hostbinding this class isn't necessary.
```suggestion
<div data-testid="ion-avatar" class="'ion-avatar">
```